test/integration/admin_can_create_store_items_test.rb

```diff
@@ -0,0 +1,23 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.